herdimmunity/output/rollingResults.txt

```diff
@@ -0,0 +1,6 @@
+As of 2021-02-04T14:02:46.445-06:00 (People Fully Vaccinated: 6436931, Date Of HerdImmunity: Saturday, August 14, 2021)
```

Comment:
What's the reasoning behind pushing rollingResults.txt and tmp.csv to the GitHub repo?

Reply:
no reason, removing it